app/tests/src/shelley.js

```diff
@@ -28,11 +29,11 @@ describe('shelley address derivation', () => {
   const mnemonic15Words =
     'abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon address'
   it('should derive base address from 15-words mnemonic', async () => {
-    const cp = await getCryptoProvider(mnemonic15Words, 0) //TODO: change discriminator to networkId for shelley
+    const cp = await getCryptoProvider(mnemonic15Words, NetworkId.TESTNET) //TODO: change discriminator to networkId for shelley
     const addrGen = ShelleyBaseAddressProvider(cp, 0, false)
     const {address} = await addrGen(0)
     const expected =
-      'addr1qzz6hulv54gzf2suy2u5gkvmt6ysasfdlvvegy3fmf969y7r3y3kdut55a40jff00qmg74686vz44v6k363md06qkq0qk0f2ud'
+      'addr_test1qzz6hulv54gzf2suy2u5gkvmt6ysasfdlvvegy3fmf969y7r3y3kdut55a40jff00qmg74686vz44v6k363md06qkq0qy0adz0'
```

Comment:
Seems strange I know but it seems the test was wrong all along, creating testnet address without the proper prefix.